server/routes/caseNotes/caseNotesController.ts

```diff
@@ -134,13 +134,15 @@ export default class CaseNotesController {
   ): Promise<void> {
     const { accessToken } = res.locals.user.token
     const { caseNoteId } = req.params
+    const backlinkPageNumber = Number(req.query.backlinkPageNumber) ?? null
```

Comment:
another nit: `Number` has some slightly weird behaviour e.g. `Number(null)` is `0`. 

i think the safest this here is to be explicit with covering all our bases. 

```
const parsedBacklinkPageNumber = Number(req.query.backlinkPageNumber) // so this either parses a string, something else, undefined, or null into a number. 
const backlinkPageNumber = parsedBacklinkPageNumber === NaN || parsedBacklinkPageNumber < 1 ? null : parsedBacklinkPageNumber
```

in this case `backlinkPageNumber` is always a positive number or null and the presenter can then handle it appropriately